tests/output.bats

```diff
@@ -29,8 +29,8 @@ load '../lib/run'
 
```

Comment:
you may want to fix this in the previous line while we are touching these tests:

```bats-mock(buildkite-agent): got buildkite-agent artifact upload docker-compose-logs/*.log
bats-mock(buildkite-agent): patterns  [2] = 'artifact' 'upload' 
bats-mock(buildkite-agent): arguments [3] = 'artifact' 'upload' 'docker-compose-logs/\*.log' 
bats-mock(buildkite-agent): unexpected argument 'docker-compose-logs/\*.log' at idx 2
bats-mock(buildkite-agent): result 0

```